apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoaderIT.java

```diff
@@ -952,8 +955,12 @@ public void loadNonBeneficiaryWhenUpdateAndFilterOnExpectRecordLoaded() {
    * the manifest includes a {@link PreValidationProperties} that is invalid per its XML Schema
    * Definition (XSD).
    *
+   * <p>Note: For (https://jira.cms.gov/browse/BFD-2222) this test will be temporarily disabled and
+   * re-enabled them once Synthea is updated for the new fields
+   *
    * @throws Exception (exceptions indicate test failure)
    */
+  @Disabled
```

Comment:
Another disabled and some defunct comments got left over here.

Reply:
On it.